internal/tracking/tracking.go

```diff
@@ -193,6 +193,7 @@ func (e *EventTracker) FlushToLogstash(ctx context.Context, cfg *config.Config,
 			CLIVersion:       versionString,
 			Host:             ioutils.GetHostname(),
 			OS:               runtime.GOOS,
+			Arch:             runtime.GOARCH,
```

Comment:
📣 TIL that `GOARCH` is set at **compile** time and not runtime, so this matches the installed build:

🔗 https://pkg.go.dev/runtime

> GOARCH, GOOS, and GOROOT are recorded at compile time and made available by constants or functions in this package, but they do not influence the execution of the run-time system.